gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/BakeController.groovy

```diff
@@ -43,9 +44,17 @@ class BakeController {
     bakeService.bakeOptions(cloudProvider)
   }
 
+  @RequestMapping(value = "/api/v1/{region}/logs/{statusId}", method = RequestMethod.GET)
```

Comment:
`/api/v1` doesn't really make sense nested under `/bakery`?

`statusId` is region-specific or global? (ie. could you do `/logs/{statusId}`)?